apps/go-mod-validator/src/deps.ts

```diff
@@ -0,0 +1,47 @@
+import { execSync } from "child_process";
+import * as core from "@actions/core";
+
+function getGoModFiles(goModDir: string): string[] {
+  let output = "";
+
+  try {
+    output = execSync(`find ${goModDir} -type f -name 'go.mod'`, {
```

Comment:
I'd recommend we use https://github.com/actions/toolkit/tree/main/packages/glob instead

Reply:
Yes that looks very good, implemented using that